components/ui/pathogen-map/pathogen-map.tsx

```diff
@@ -142,7 +142,7 @@ export function PathogenMap<
       minZoom={2}
       maxZoom={14}
       interactiveLayerIds={[...layers.map((layer) => layer.id), 'country-highlight-layer']}
-      mapboxAccessToken={process.env.NEXT_PUBLIC_MAPBOX_API_KEY as string}
+      mapboxAccessToken={process.env.MAPBOX_API_KEY as string}
```

Comment:
Vercel preview not working for me :(

![image](https://github.com/serotracker/dashboard/assets/86806388/87dd8cd5-10d4-4574-b874-bd2a19a44d36)

Reply:
We probably have to figure out something a bit more clever sadly :/

Reply:
Oh also just checked vercel, I would def make sure this is set in production too.

Reply:
yeah just saw that damn... ok will take a look

Reply:
I guess it wasnt ezpz lmnsqzy